baseclass-samples/java-callouts-baseclass-exception/apiproxy/proxies/default.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<ProxyEndpoint name="default">
+    <Description/>
+    <FaultRules/>
+    <PreFlow name="PreFlow">
+        <Request>
```

Comment:
Super nit: You could make this <Request/> since there's nothing that happens for the request.